pipeline/util.sh

```diff
@@ -197,21 +197,27 @@ REGION_KEY['ap-southeast-1']='singapore-key-benchmark.pem'
 REGION_KEY['eu-central-1']='frankfurt-key-benchmark.pem'
 REGION_KEY['eu-west-1']='ireland-key-benchmark.pem'
 REGION_KEY['gcp']='harmony-node.pem'
+REGION_KEY['do']='harmony-node.pem'
```

Comment:
better use a different keypair

Reply:
generated a new key pair for DO. name: do-node.pem